<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#statusbar{
background-color:rgb(0, 0, 127);color: rgb(255, 255, 255);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 5, 68, 255), stop:1 rgba(10, 55, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 87 18pt &quot;Arial&quot;;
 border-bottom: 5px solid black;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;CATEGORIES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_field">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>621</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
color: rgb(255, 255, 255);
font: italic 18pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="business">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>BUSINESS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entertainment">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>ENTERTAINMENT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="general">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>GENERAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="health">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>HEALTH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="science">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>SCIENCE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sports">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>SPORTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="technology">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>TECHNOLOGY</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/search/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="proposed_content">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>710</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>PROPOSED CONTENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: italic 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>LOG IN</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chBx_business">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chBx_entertainment">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chBx_general">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chBx_health">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chBx_science">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chBx_sports">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chBx_technology">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>691</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);background-color: rgba(0,0,0,0);
font: 12pt &quot;Arial&quot;;
margin-bottom: 12px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>620</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="search.qrc"/>
 </resources>
 <connections/>
</ui>
